mitreattack/navlayers/core/technique.py

```diff
@@ -149,6 +164,8 @@ def _loader(self, data):
                 self.metadata = data[entry]
             elif entry == 'showSubtechniques':
                 self.showSubtechniques = data[entry]
+            elif entry == 'aggregateScore':
+                self.aggregateScore = data[entry]
```

Comment:
Some of your test scripts had it manually set, so I left in the ability to import it. If it is, it will still always be calculated on the fly for exports. Still want me to change it?

Reply:
Yeah I think some of the test layers were generated on an earlier iteration of the feature where they were included in layers. It should not be included as of the final Navigator version for this release.

Reply:
Gotcha, I'll remove it.